Components/cards/cards.css

```diff
@@ -0,0 +1,182 @@
+body {
+  font-family: sans-serif;
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.card-basic {
+  width: 20rem;
+  border-radius: 4px;
+  box-shadow: 0 2px 1px -1px rgb(0 0 0 / 20%), 0 1px 1px 0 rgb(0 0 0 / 14%),
+    0 1px 3px 0 rgb(0 0 0 / 12%);
+  background-color: white;
+}
+
+.card-img {
+  width: 100%;
+  height: 12rem;
+  object-fit: cover;
+  border-radius: 4px 4px 0 0;
+}
+
+.card-heading {
+  margin-left: 12px;
+  margin-bottom: 4px;
+  letter-spacing: 0.025rem;
+  color: #393d3d;
+}
+
+.card-subHeading {
+  margin-left: 12px;
+  letter-spacing: 0.03rem;
+  color: #6a6e6e;
+  font-weight: 500;
+}
+
+.card-description {
+  margin-left: 12px;
```

Comment:
using rem is preferred i think that makes ur whole code consistent

Reply:
yess